.github/workflows/ci.yml

```diff
@@ -38,15 +38,12 @@ jobs:
           find ./ -name "__pycache__" -type d -print0 |xargs -0 rm -rf
           find ./ -name "*.pyc" -type f -print0 |xargs -0 rm -rf
           find ./ -name "*.so" -type f -print0 |xargs -0 rm -rf
-          pip install -U setuptools pip
-          pip install -U --upgrade-strategy=eager -r requirements.txt
-          pip install -U --upgrade-strategy=eager -r setup_requirements.txt
+          pip install -U setuptools pip wheel
           pip install -U --upgrade-strategy=eager -r test_requirements.txt
```

Comment:
Can we get rid of `test_requirements.txt `?

There is a solution by adding `extras_require` with a `test` entry (with same value as `tests_require`).  The `pip install -e '.[test]'` command can be used.

Reply:
Sure.

Fyi, I'm not planning to merge this until I move to PEP517. I'll change it to a draft.